pkg/pgmodel/pgx.go

```diff
@@ -573,6 +590,49 @@ func (h *insertHandler) flushPending(pending *pendingBuffer) {
 			copyColumns,
 			&pending.batch,
 		)
+		if err != nil {
```

Comment:
I think this block should be moved into its own function

Reply:
moved part of it